urllib3/poolmanager.py

```diff
@@ -112,6 +113,8 @@ class PoolManager(RequestMethods):
 
     def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+        if "transport_security_manager" not in connection_pool_kw:
+            connection_pool_kw["transport_security_manager"] = TransportSecurityManager()
```

Comment:
Sounds good. OK, I added it to HTTPPoolKey and added a unit test. Please take a look.